flags.go

```diff
@@ -309,6 +309,12 @@ func newApp() (app *cli.App) {
 					"When this flag is specified, GCSFuse uses the GCS JSON API instead.",
 			},
 
+			cli.BoolTFlag{
+				Name: "enable-managed-folders",
+				Usage: "GCSFuse automatically lists managed folders found within a bucket. " +
```

Comment:
We can enhance the usage statement. We should also say about the `--implicit-dirs` flag something like - customer must pass `--implicit-dirs` flag to make this flag work for true value.